src/main/cli/cli.c

```diff
@@ -269,6 +269,25 @@ static const char * const *sensorHardwareNames[] = {
 };
 #endif // USE_SENSOR_NAMES
 
+// Needs to be aligned with mcuTypeId_e
+static const char *mcuTypeNames[] = {
+    "SIMULATOR",
+    "F103",
+    "F303",
+    "F40X",
+    "F411",
+    "F446",
+    "F722",
+    "F745",
+    "F746",
+    "F765",
+    "H750",
+    "H743 (Rev Unknown)",
+    "H743 (Rev.Y)",
+    "H743 (Rev.X)",
+    "H743 (Rev.V)",
+};
```

Comment:
Can we justify the extra space claimed by members that are never used in a specific build?

Reply:
Turns out that since access to these elements is only done in constant expressions (except for H743), the compiler is already taking care of eliminating unused elements:

```
mikeller@pc:~/git/betaflight$ strings obj/betaflight_4.2.0_STM32F405_norevision.bin | grep F4
STM32F405
% F4
	N=F4F
F4I@F
B%F4p
Betaflight STM32F405
STM32F405
STM32F405
F40X
mikeller@pc:~/git/betaflight$ strings obj/betaflight_4.2.0_STM32F405_norevision.bin | grep F7
F3F0F7F
Ju"F7
F7F F
mikeller@pc:~/git/betaflight$ strings obj/betaflight_4.2.0_STM32F7X2_norevision.bin | grep F722
F722
mikeller@pc:~/git/betaflight$ strings obj/betaflight_4.2.0_STM32F7X2_norevision.bin | grep F4
L F4
d F4
MF4K
N=F4F
BF4I(F
```

If we are worried about flash space usage on H743 we could always address this by wrapping the strings in conditionals like so

```
static const char *mcuTypeNames[] = {
#if defined(SIMULATOR_BUILD)
￼    "SIMULATOR",
#else 
    NULL,
#endif
￼    [...]
```

But this would turn the array real ugly, and we are not doing this for any of the other constant string arrays in `cli.c`.